docs/README.md

```diff
@@ -215,356 +153,385 @@ for these executables in your `$PATH`.
+▹▸▹▹▹ 🚀 Deploying...
+{
+  "assets": [
+    {
+      "digest": "79e74d6a7b68a35aeb9aa2dd7f6083dae5fdba5b6a2f199529b6c49624d1e27b",
+      "id": 1,
+      "index_id": 1,
+      "version": 1
+    },
+    {
+      "digest": "4415628d9ea79b3c3f1e6f02b1af3416c4d0b261b75abe3cc81b77b7902549c5",
+      "id": 1,
+      "index_id": 1,
+      "version": 1
+    },
+    {
+      "digest": "e901eba95ce8b4d1c159c5d66f24276dc911e87dbff55fb2c10d8b371528eacc",
+      "id": 1,
+      "index_id": 1,
+      "version": 1
     }
+  ],
+  "success": "true"
 }
+▪▪▪▪▪ ✅ Successfully deployed index.
+```
+
+### 3. Querying for data
+
+With our index deployed, after a few seconds, we should be able to query for newly indexed data.
+
+Below, we write a simple GraphQL query that simply returns a few fields from all transactions that we've indexed.
+
+```bash
+curl -X POST http://0.0.0.0:29987/api/graph/my_project \
+   -H 'content-type: application/json' \
+   -d '{"query": "query { tx { id hash status block }}", "params": "b"}' \
+| json_pp
+```
+
+```text
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100   364  100   287  100    77   6153   1650 --:--:-- --:--:-- --:--:--  9100
+[
+   {
+      "block" : 7017844286925529648,
+      "hash" : "fb93ce9519866676813584eca79afe2d98466b3e2c8b787503b76b0b4718a565",
+      "id" : 7292230935510476086,
+      "status" : {
+         "block" : "0x8c34daaa2c58629cb98fa66d4f5ce0c0850d24e655ed6006b22204dac42fd918",
+         "status" : "success",
+         "time" : "2022-11-10 14:35:58 UTC"
+      }
+   },
+   {
+      "block" : 3473793069188998756,
+      "hash" : "5ea2577727aaadc331d5ae1ffcbc11ec4c2ba503410f8edfb22fc0a72a1d01eb",
+      "id" : 4136050720295695667,
+      "status" : {
+         "block" : "0x2b892dd6574e4a803f90c85754d6c8e154ec5f7dd91a25ce962820dce12f15e5",
+         "status" : "success",
+         "time" : "2022-11-10 13:35:58 UTC"
+      }
+   },
+   {
+      "block" : 7221293542007912803,
+      "hash" : "d2f638c26a313c681d75db2edfbc8081dbf5ecced87a41ec4199d221251b0578",
+      "id" : 4049687577184449589,
+      "status" : {
+         "block" : "0xa0812af8738da14f7db2f00a53341492aa339f8d88e118820e78a500b11e3560",
+         "status" : "success",
+         "time" : "2022-11-10 12:35:58 UTC"
+      }
+   },
+]
+```
+
+#### Finished! 🥳
 
+Congrats, you just created, built, and deployed your first index on the world's fastest execution layer. For more detailed info on how the Fuel indexer service works, make sure you [**read the book**](https://fuellabs.github.io/fuel-indexer/master/).
+
+## Plugin
+
+The primary way of developing Fuel indexers for users that are not also contributors, is via the `forc-index` plugin. The `forc-index` plugin, is a CLI tool that is bundled with Fuel's primary CLI tooling interface, [`forc`](https://github.com/FuelLabs/sway/tree/master/forc) ("Fuel Orchestrator").
```

Comment:
Nit: Remove the commas after contributors and plugin.

Also, I think it's more clear if we change "users that are not also contributors" to "end users". In my estimation, an end user is usually considered to be different from a contributor on most projects, so I think it would be fine here.